stable/anchore-engine/templates/catalog_deployment.yaml

```diff
@@ -254,3 +254,29 @@ spec:
   selector:
     app: {{ template "anchore-engine.fullname" . }}
     component: {{ $component }}
+
+{{- if .Values.anchoreCatalog.runtimeInventory.reportAnchoreCluster.enabled }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: catalog-cluster-read-only
+rules:
+- apiGroups: [""]
+  resources: ["pods","namespaces"]
+  verbs: ["get", "watch", "list"]
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: catalog-cluster-read-only-binding
+subjects:
```

Comment:
@dakaneye I believe that this is going to need a service account to bind to, which the chart doesn't currently create.

Reply:
The chart does let you specify a service account to run deployments as, but nothing in the chart actually creates the service account. I think that if we have a thing that _requires_ a service account, we should create it in the chart.

Reply:
I added one for the catalog specifically!